sympy/physics/continuum_mechanics/beam.py

```diff
@@ -65,34 +66,37 @@ class Beam(object):
          + Piecewise(((x - 2)**4, x - 2 > 0), (0, True))/4)/(E*I)
     """
 
-    def __init__(self, length, elastic_modulus, second_moment, variable=Symbol('x'), base_char='C'):
+    def __init__(self, length, elastic_modulus, shape, variable=Symbol('x'), base_char='C'):
```

Comment:
Don't change the argument name in case people called it explicitly. `second_moment=I`. This retains backwards compatibly.